frontend/app/src/pages/settings/api-keys/external/index.vue

```diff
@@ -58,10 +65,12 @@ const updateKeys = ({
   beaconchain,
   loopring,
   opensea,
-  optimismEtherscan
+  optimismEtherscan,
+  polygonPosEtherscan
 }: ExternalServiceKeys) => {
   evmEtherscanTabs.ethereum.value = etherscan?.apiKey || '';
   evmEtherscanTabs.optimism.value = optimismEtherscan?.apiKey || '';
+  evmEtherscanTabs.polygon_pos.value = polygonPosEtherscan?.apiKey || '';
```

Comment:
Do you think it would be easy to make sure we don't have snake_case in the frontend code for consistency?

Reply:
checking, perhaps i'd just have to use the Blockchain enum here.